Backpack/src/main/res/values/themes.xml

```diff
@@ -10,6 +10,10 @@
     <item name="windowActionBar">false</item>
     <item name="colorButtonNormal">@android:color/white</item>
 
+    <item name="bpkFontFamilyBase">sans-serif</item>
```

Comment:
we do need to define some config for fonts, so you can use the library - these can be overridden by apps